src/main/java/Task4.java

```diff
@@ -1,5 +1,7 @@
-public class Main {
+public class Task4 {
```

Comment:
Did we want to provide the starter code that was shown in the README?

Reply:
I updated the task4 instructions to make adding the statements shown in the readme part of the unordered list.  Does that clarify?  I am leaving the main in Task4.java empty to avoid giving code with compiler errors.

Reply:
Yes it does! I figured the reason was to avoid the compiler errors! But wanted to clarify. Thanks!